assignments/objects.js

```diff
@@ -18,20 +18,58 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
+  "id": 1,
+  "name" : "Mitzi",
+  "email":"mmelloy0@psu.edu",
+  "gender": "F"
+}
+const Kennan = {
+  "id": 2,
+  "name" : "Kennen",
+  "email":"kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const Keven = {
+  "id": 3,
+  "name" : "Keven",
+  "email":"kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const Gannie = {
+  "id": 4,
+  "name" : "Gannie",
+  "email":"gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const Antonietta = {
+  "id": 5,
+  "name" : "Antonietta",
+  "email":"adaine5@samsung.com",
+  "gender": "F"
+}
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
```

Comment:
in the directions he is asking you to log out this means to do a console.log  make sure you read everything

Reply:
// Mitzi's name
console.log(mitzi.name);
// Kenna's ID
console.log(kennan.id);
// Keven's email
console.log(keven.email);
// Gannie's name
console.log(gannie.name);
// Antonietta's Gender
console.log(antonietta.gender);